plugin/entryBase.go

```diff
@@ -21,18 +19,11 @@ const (
 
 var defaultOpCodeToNameMap = [3]string{"List", "Open", "Metadata"}
 
-type syncedAttribute struct {
-	*SyncableAttribute
-	key string
-}
-
 // EntryBase implements Entry, making it easy to create new entries.
 // You should use plugin.NewEntry to create new EntryBase objects.
 type EntryBase struct {
 	entryName          string
-	attrMux            *sync.RWMutex
 	attr               EntryAttributes
```

Comment:
I considered exporting the field, but that breaks consistency with the other `Set*` methods so chose not to.